File: java/URI_1017.java
package URI;
import java.util.Locale;
import java.util.Scanner;

public class URI_1017 {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int tempoViagem = sc.nextInt();
		int veloMedia = sc.nextInt();
		
		double litros = veloMedia * tempoViagem / 12.0;
		
		System.out.printf("%.3f%n", litros);
		
		sc.close();

	}

}

File: kotlin/URI_1017.kt
import java.util.*

fun main(args: Array<String>) {
    Locale.setDefault(Locale.US)
    val sc = Scanner(System.`in`)
    val tempo = sc.nextInt()
    val velocidade = sc.nextInt()
    println("%.3f".format(tempo*velocidade/12.0))
}